build-system/tasks/clean.js

```diff
@@ -3,75 +3,80 @@
 const argv = require('minimist')(process.argv.slice(2));
 const del = require('del');
 const fs = require('fs-extra');
+const globby = require('globby');
 const path = require('path');
 const {cyan, yellow} = require('kleur/colors');
+const {ignoreFile, splitIgnoreListByHeader} = require('./check-ignore-list');
 const {log} = require('../common/logging');
 
 const ROOT_DIR = path.resolve(__dirname, '../../');
 
+/**
+ * @return {Promise<string[]>}
+ */
+async function getPathsToDelete() {
```

Comment:
Since this is no longer returning the entire set of paths to delete, I think a better name for this function would be `getIgnoreList`.